<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_workflow</class>
 <widget class="QDialog" name="Dialog_workflow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>870</width>
    <height>800</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>870</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>20</y>
     <width>341</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;img src=&quot;:/images/GUI_icons/logo_transparent.png&quot;/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>150</y>
     <width>767</width>
     <height>32</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="pushButton_cleaning_step">
      <property name="text">
       <string>Input Data / Pre-processing</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_assembling_step">
      <property name="text">
       <string>Assembly</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_mapping">
      <property name="text">
       <string>Mapping / Alignment</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_marker_step">
      <property name="text">
       <string>Marker Discovery / Selection</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_workflow">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>830</width>
     <height>570</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>830</width>
     <height>570</height>
    </size>
   </property>
   <property name="frameShape">
    <enum>QFrame::Box</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>label</zorder>
  <zorder>label_workflow</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
